packages/hoc-disabled-tooltip/src/styledComponents/Wrapper.js

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+
+const Wrapper = styled.div`
+  display: inline-block;
```

Comment:
this can break the ``fluid`` option on the button component

Reply:
The reason this inline block exists:

Not all form inputs can trigger the onMouseOver and onMouseLeave events, ex: a disabled checkbox input, buttons, etc. So one way to display tooltip on mouseover is to have a hit area overlay on top of the wrapped component. And since we dont know the width of the element, we use a container with inline style to host both the wrapped component and t he hit area.

Also, the main use of this hoc is for inline elements such as links, buttons and form inputs...

But this can break the expected layout widths of block elements witth width set in percents, like a button with 100% width (the case of a farmblocks button with the fluid flag set).

I dont know of an elegant way to handle that case, one simple solution was to have a new property to cover this exception and remove the default display inline behaviour when needed.

For this moment I am submitting a patch that implements a new ``displayBlock`` property. I am not super fan of this idea and I am open to suggestions, either on a separate issue or on this same thread.

Reply:
the text-input previous default was display block, so I am thinking in reversing this again, make display block be the default, remove the displayBlock property and leave it for the user to wrap it in another div if she needs specific widths.

This will be included in the https://github.com/CraveFood/farmblocks/pull/171 patch

Reply:
instead of reversing, I ended up just adding the displayBlock in the properties internally on th e input-text refactorying...